rotkehlchen/utils/validation_schemas.py

```diff
@@ -0,0 +1,24 @@
+from marshmallow import Schema, fields
+
+from rotkehlchen.api.v1.encoding import UnderlyingTokenInfoSchema
```

Comment:
This should not be under the general `utils/` directory. As I mentioned above in the other comment that you should make a class/module for all this logic. This should go in the same place.